tribuo-classification/client/src/main/java/com/r3/conclave/sample/client/Client.java

```diff
@@ -28,114 +26,112 @@
  * trained model can be used to make predictions of a given input tumor.
  *
  */
-public class Client {
-    private static final String TRAIN = "TRAIN";
+@CommandLine.Command(name = "psi-client",
+        mixinStandardHelpOptions = true,
+        description = "Simple client that communicates with the PSIEnclave using the web host.")
+public class Client implements Callable<Void> {
+
+    @CommandLine.Option(names = {"-u", "--url"},
+            required = true,
+            description = "URL of the web host running the enclave.")
+    private String url;
+
+    @CommandLine.Option(names = {"-c", "--constraint"},
+            required = true,
+            description = "Enclave constraint which determines the enclave's identity and whether it's acceptable to use.",
+            converter = EnclaveConstraintConverter.class)
+    private EnclaveConstraint constraint;
+
+    @CommandLine.Option(names = {"-f", "--file"},
+            required = true,
+            description = "URL of the web host running the enclave.")
+    private String fileName;
```

Comment:
This variable name is a bit too vague. What is this is a file of?

Reply:
This file contains the input training data for the AI model. Updated the name and description.